pcdsdevices/tags.py

```diff
@@ -0,0 +1,13 @@
+tag_to_explanation = {
+    'protected': 'May require additional privileges to change.',
+}
+
+
+def get_valid_tags():
+    """Return a list of all valid variety tags."""
+    return list(tag_to_explanation)
```

Comment:
I know it's a small thing, but for some reason this collection of tags being a list and every other collection of tags being a set busts my brain

Reply:
No, that's definitely valid; fixed up in bada93c